IdentityCore/src/parameters/MSIDInteractiveRequestParameters.h

```diff
@@ -40,6 +40,7 @@
 #if TARGET_OS_IPHONE
 @property (nonatomic) UIViewController *parentViewController;
 @property (nonatomic) UIModalPresentationStyle presentationType;
+@property (nonatomic) BOOL prefersEphemeralWebBrowserSession API_AVAILABLE(ios(13.0));
```

Comment:
would this also be applicable for macOS now?

Reply:
yes, but MacOS will be addressed in separate PR.